### ISSUE TITLE
find_negative_cycle() raises Exception when passed a list as the source argument

### ISSUE THREAD

**Issue by Bumblebee118** [networkx/networkx#5610]
The problem of this issue has also been discussed in [this StackOverflow post.](https://stackoverflow.com/questions/72142592/networkx-find-negative-cycle-parameters?noredirect=1#comment127468735_72142592)

### Current Behavior
When a list of nodes is passed as an argument into the find_negative_cycle() function the following error occurs:
`TypeError: unhashable type: 'list'`

### Expected Behavior
No error should happen and a lists of nodes containing the negative cycle should be returned.

### Steps to Reproduce
Generate a graph with a negative weight cycle.
execute the following command: nx.find_negative_cycle(graph, [0])

### Environment
Python version: 3.9
NetworkX version: 2.8

**Comment by dschult** [comment]
Yes, this is a documentation error. The parameter should not be a list. 
It should be a node from which to start the search for a negative cycle. 
The documentation is wrong.   Thank you for bringing this to our attention.
